hivemq-edge/src/main/java/com/hivemq/embedded/internal/EmbeddedHiveMQImpl.java

```diff
@@ -213,6 +213,7 @@ private void performStop(
 
             try {
                 hiveMQServer.stop();
+                hiveMQServer.shutdownProtocolAdapters();
```

Comment:
does the order matter? i.e. could/should hiveMQServer.stop(); happen after hiveMQServer.shutdownProtocolAdapters(); instead?

Reply:
it actually matters. Fixed.